pages/establishment/formatters/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { capitalize } from 'lodash';
+
+const listFormatter = list => {
+  if (list.length < 1) {
+    return 'None selected';
+  }
+
+  return list.map(item => item === 'procedure' ? 'Procedures' : capitalize(item)).join(', ');
```

Comment:
Where is this used? It seems odd to not use the full wording for these options.

Reply:
These are used in the side-by-side diff, so the full wording is awkward.